### ISSUE TITLE
Question for hyperparameters 

### ISSUE THREAD

**Issue by zy-fang** [JinmiaoChenLab/GraphST#6]
Hi Jinmiao,

I am interested in your method GraphST. It is a nice method for analyzing spatial transcriptomics! 

I tried to cluster DLPFC samples with GraphST (follow this code "https://deepst-tutorials.readthedocs.io/en/latest/Tutorial%201_10X%20Visium.html"). I found that some of the samples matched the results described in your paper. But some of the samples are quite different from the results in the paper. I wonder if the hyperparameter Settings are inconsistent. I would like you to provide a list of hyperparameters for each sample.

Best,
Zhaoyu

**Comment by zy-fang** [comment]
Hi Jinmiao,

I am interested in your method GraphST. It is a nice method for analyzing spatial transcriptomics! 

I tried to cluster DLPFC samples with GraphST (follow this code "https://deepst-tutorials.readthedocs.io/en/latest/Tutorial%201_10X%20Visium.html"). I found that some of the samples matched the results described in your paper. But some of the samples are quite different from the results in the paper. I wonder if the hyperparameter Settings are inconsistent. I would like you to provide a list of hyperparameters for each sample.

Best,
Zhaoyu

**Comment by longyahui** [comment]
Hi Zhaoyu,

    Thank you for your interest in our work. 

    For all clustering results in our paper, we use the same hyper-parameters. 

    One possible reason may be random seed issue. Our GraphST model is involved in random seed. All results were obtained based on NVIDIA RTX A6000 GPU. Different computers or GPUs may generate different random values despite the same setting of random seed.

    Could you provide more details for further analysis, please? My email is long_yahui@immunol.a-star.edu.sg. Thanks again.

    Best regards,
    Yahui

**Comment by KalinNonchev** [comment]
I am experiencing the same. @zy-fang could you share your results?
<img width="717" alt="image" src="https://user-images.githubusercontent.com/50597791/230171969-a07ecaf1-1b98-49be-904e-3a94116cade0.png">
